README.md

```diff
@@ -184,6 +184,34 @@ cache_fragment(post)
 cache_fragment(post) { post }
 ```
 
+Using literals: Even when using a same string for all queries, the cache changes per argument because of the query_key.
```

Comment:
```suggestion
Using literals: Even when using a same string for all queries, the cache changes per argument and per selection set (because of the query_key).
```